### ISSUE TITLE
Add disabled option to checkboxes

### ISSUE THREAD

**Issue by jrbarnes9** [ONSdigital/design-system#2141]
The use of checkboxes in filters requires the addition of a disabled state for items which return zero result counts.

- `$Grey35` can safely be used for the checkbox and label as [WCAG2.1 states](https://www.w3.org/TR/WCAG21/#contrast-minimum) "disabled interface need not meet contrast minimum" (see incidental)
- `aria-disabled` could be applied to the `label` and `input` to supplement the standard `<input ... disabled />` [MDN reference](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-disabled)
- cursor should be set to `cursor: not-allowed;` for label and input.

As seen in 

![Publications filter selected (disabled zero results)](https://user-images.githubusercontent.com/22910336/163198948-3455e1cc-6485-4237-8a90-375da1c35a07.png)

and
<img width="197" alt="Screenshot 2022-04-13 at 11 09 40" src="https://user-images.githubusercontent.com/22910336/163197697-571e75be-04c1-4d8c-ba4d-f84c34087c78.png">

@jondewijones following this item for use in search.

_Originally posted by @armstrongb in https://github.com/ONSdigital/design-system/discussions/13#discussioncomment-2559997_

**Comment by jrbarnes9** [comment]
@yatesn New request from DP

**Comment by jrbarnes9** [comment]
Resolved in #2187
